src/BugAtlasServiceProvider.php

```diff
@@ -21,8 +21,14 @@ class BugAtlasServiceProvider extends ServiceProvider
     public function boot(Request $request)
     {
         $this->mergeConfigFrom(__DIR__ . '/../config/bugatlas.php', 'bugatlas');
-        $logDetails = $this->prepareLogDetails($request);
-        $this->sendLogToApi($logDetails);
+
+        // Check if the required keys are available in the merged configuration
+        if (!empty(config('bugatlas.api_key')) && !empty(config('bugatlas.secret_key')) && !empty(config('bugatlas.tag'))) {
```

Comment:
Change the logic to check the values are empty, if empty return false.

Reply:
comments resolved